acceptance/tests/sameness/main_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package sameness
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	testsuite "github.com/hashicorp/consul-k8s/acceptance/framework/suite"
+)
+
+var suite testsuite.Suite
+
+func TestMain(m *testing.M) {
+	suite = testsuite.NewSuite(m)
+
+	expectedNumberOfClusters := 4
+
+	if suite.Config().EnableMultiCluster && suite.Config().IsExpectedClusterCount(expectedNumberOfClusters) && suite.Config().UseKind {
+		os.Exit(suite.Run())
+	} else if suite.Config().EnableMultiCluster && suite.Config().IsExpectedClusterCount(expectedNumberOfClusters) {
+		fmt.Println(fmt.Sprintf("Skipping sameness tests because either -enable-multi-cluster is "+
+			"not set or the number of clusters did not match the expected count of %d", expectedNumberOfClusters))
```

Comment:
It seems the only difference between this and the prior conditional is the first one also checks `&& suite.Config().UseKind`. I think maybe the `&&` here should be an `||`?

Reply:
hmm yeah, I think I was initially trying to make the logging more meaningful, but I can do that with a single log. I've replaced it with a single `if...else` with:

```Go
		fmt.Println(fmt.Sprintf("Skipping sameness tests because either -enable-multi-cluster is "+
			"not set, the number of clusters did not match the expected count of %d, or --useKind is false. "+
			"Sameness acceptance tests are currently only suopported on Kind clusters", expectedNumberOfClusters))
```

Reply:
That seemed reasonable to me! Just the logic seemed slightly off - but either way, sounds good 👍🏻